packages/@react-native/repo-config/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@react-native/repo-config",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Meta package for the React Native repository.",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:facebook/react-native.git"
+  },
+  "dependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/generator": "^7.5.0",
+    "@reactions/component": "^2.0.2",
+    "async": "^2.4.0",
+    "babel-eslint": "^10.1.0",
+    "clang-format": "^1.2.4",
+    "connect": "^3.6.5",
+    "coveralls": "^3.0.2",
+    "detox": "15.4.4",
```

Comment:
I was a bit concerned adding this as a package into our repo would bloat things, but the extra tree of dependencies was less bad than I expected.